src/extension.ts

```diff
@@ -25,7 +25,6 @@ export async function activate(context: vscode.ExtensionContext) {
     commandServerApi: () => commandServerApi,
     getNodeAtLocation: () => getNodeAtLocation,
   } as FactoryMap<Graph>);
-  graph.debug.init();
```

Comment:
`Debug#init` was empty, so removed it altogether, since it's not implementing anything.

Reply:
The way the graph works, accessing an attribute will cause the component to be instantiated, so it's possible this line was actually doing something

Reply:
I believe this is the only remaining issue on this PR

Reply:
Reverted this change. I'm not convinced this line is doing much in practice, but we can look into it later.